fontforgeexe/bitmapview.c

```diff
@@ -779,6 +779,7 @@ static void BVExpose(BitmapView *bv, GWindow pixmap, GEvent *event ) {
 	for ( refs = cvtemp->b.sc->layers[ly_fore].refs; refs!=NULL; refs = refs->next )
 	    CVDrawSplineSet(cvtemp,pixmap,refs->layers[0].splines,col,false,&clip);
     }
+    CVUnlinkView(cvtemp);
     CharViewFree(cvtemp);
```

Comment:
This still isn't perfect but at least it no longer crashes. However&hellip;it leads to a phantom `fontforge` process. I remember this happened before when we were talking about modals. Do you know why this might be happening @jtanx and how I can fix it?

Reply:
Sorry, it's getting late, I think I'll only be able to look at this tomorrow, unless someone else wants to take a look. Just noting I find this a bit odd, so I'd like to get a better understanding of why it was fine to not allocate a charview before/why it's needed now, and why these frees are causing issues.

Reply:
No problem :-)